app/models/user.rb

```diff
@@ -46,6 +47,10 @@ def activate
     update_columns activated: true, activated_at: Time.zone.now
   end
 
+  def update_activated user
+    user.update_column :activated, !user.activated
```

Comment:
```suggestion
    update_column :activated, true
```
ri luôn cho rồi, nghịch đảo đồ
hay em có chức năng actice rồi un actve rồi active... hehe